lib/update_security_release.js

```diff
@@ -268,17 +269,26 @@ Summary: ${summary}\n`,
   async calculateVersions(affectedVersions, supportedVersions) {
     const h1AffectedVersions = [];
     const patchedVersions = [];
+    let isPatchRelease = true;
     for (const affectedVersion of affectedVersions) {
-      const major = affectedVersion.split('.')[0];
-      const latest = supportedVersions.find((v) => v.major === Number(major)).version;
+      const affectedMajor = affectedVersion.split('.')[0];
+      const latest = supportedVersions.find((v) => v.major === Number(affectedMajor)).version;
       const version = await this.cli.prompt(
         `What is the affected version (<=) for release line ${affectedVersion}?`,
         { questionType: 'input', defaultAnswer: latest });
 
-      const nextPatchVersion = parseInt(version.split('.')[2]) + 1;
+      const nextPatchVersion = semver.inc(version, 'patch');
+      const nextMinorVersion = semver.inc(version, 'minor');
       const patchedVersion = await this.cli.prompt(
         `What is the patched version (>=) for release line ${affectedVersion}?`,
-        { questionType: 'input', defaultAnswer: nextPatchVersion });
+        {
+          questionType: 'input',
+          defaultAnswer: isPatchRelease ? nextPatchVersion : nextMinorVersion
```

Comment:
Nit: Let’s not compute both if we’re using only one
```suggestion
          defaultAnswer: semver.inc(version, isPatchRelease ? 'patch' : 'minor')
```